.github/workflows/pkgci_regression_test.yml

```diff
@@ -313,3 +327,25 @@ jobs:
             --log-cli-level=info \
             --retries 7
           echo "$(<job_summary.md )" >> $GITHUB_STEP_SUMMARY
+          rm job_summary.md
+
+      - name: "Running SDXL rocm pipeline benchmark (mi300)"
+        if: contains(matrix.name, 'rocm_mi300_gfx942')
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest SHARK-TestSuite/iree_tests/benchmarks/sdxl/benchmark_sdxl_rocm.py \
+            --goldentime-rocm-e2e-ms 320 \
+            --goldentime-rocm-unet-ms 76 \
+            --goldentime-rocm-clip-ms 14.5 \
```

Comment:
This is still 3 iterations of unet right? Do we want to run more iterations? I think we've been benchmarking in other places with 20 or so? If we can spare the CI time then getting "real" / "directly comparable" numbers would be nice.

Reply:
Yeah it's at 3 right now, just so we can benchmark across backends (cpu, rocm, vulkan) without having to take too long. I don't think it's too important to match the exact number of steps, but that is something we can change fairly easily

Reply:
I don't have a particularly strong opinion. Let's just keep the methodology documented so there isn't any confusion about what we're measuring and why.